pkg/config/generate.go

```diff
@@ -510,10 +534,10 @@ func randomBytes(n int) ([]byte, error) {
 	return b, nil
 }
 
-func randomStorageAccountName() (string, error) {
-	const letterBytes = "abcdefghijklmnopqrstuvwxyz0123456789"
+func randomString(lenght int) (string, error) {
```

Comment:
reluctant to keep code which does 90% the same. If a storage account name can live with capital letters in the name, this simplifies things a little bit.